### PR TITLE
Fix javadoc errors and warnings in idlj module

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
@smillidge @russgold could you review this please?

File: idlj/src/main/java/com/sun/tools/corba/ee/idl/PragmaHandler.java [hunk 1]
```diff
@@ -55,13 +62,18 @@ protected String getStringToEOL () throws IOException
       character.  It encapsulates parenthesis and quoted strings, meaning
       it does not stop if the given character is found within parentheses
       or quotes.  For instance, given the input of `start(inside)end',
-      getUntil ('n') will return "start(inside)e" */
+      getUntil ('n') will return "start(inside)e"
```
Comment (russgold) [inline idlj/src/main/java/com/sun/tools/corba/ee/idl/PragmaHandler.java]: Should have a period after start(inside)e

File: idlj/src/main/java/com/sun/tools/corba/ee/idl/PragmaHandler.java [hunk 2]
```diff
@@ -55,13 +62,18 @@ protected String getStringToEOL () throws IOException
...
+    * @param c token to read up to. The token itself will not be read
+    */
```
Comment (russgold) [inline idlj/src/main/java/com/sun/tools/corba/ee/idl/PragmaHandler.java]: should have a period after "be read"

File: idlj/src/main/java/com/sun/tools/corba/ee/idl/Preprocessor.java [hunk 3]
```diff
@@ -908,6 +912,7 @@ String getStringToEOL () throws IOException
    * meaning it does not stop if the given character is found within
    * parentheses or quotes.  For instance, given the input of
    * `start(inside)end', getUntil ('n') will return "start(inside)e"
+   * @param c token to read up to. The token itself will not be read
```
Comment (russgold) [inline idlj/src/main/java/com/sun/tools/corba/ee/idl/Preprocessor.java]: need period after "be read"

File: idlj/src/main/java/com/sun/tools/corba/ee/idl/UnionEntry.java [hunk 4]
```diff
@@ -108,7 +114,7 @@ public void addContained (com.sun.tools.corba.ee.idl.SymtabEntry entry)
     _contained.addElement (entry);
   } // addContained
 
-  /** This is a vector of SymtabEntry's.  It itemizes any types which
+  /** This is a vector of SymtabEntrys.  It itemizes any types which
```
Comment (russgold) [inline idlj/src/main/java/com/sun/tools/corba/ee/idl/UnionEntry.java]: For consistency (with the very next edit), please use the same @link notation.

File: idlj/src/main/java/com/sun/tools/corba/ee/idl/ValueEntry.java [hunk 5]
```diff
@@ -71,19 +71,25 @@ public void generate (Hashtable symbolTable, PrintWriter stream)
...
+    * return a vector which only contains InterfaceEntry's.
+    * @param supports a {@link ForwardEntry} or {@link InterfaceEntry}
+    */
   public void addSupport (com.sun.tools.corba.ee.idl.SymtabEntry supports)
   {
     _supports.addElement (supports);
   } // addSupport
 
-  /** This method returns a vector of InterfaceEntry's. */
+  /** This method returns a vector of InterfaceEntry's.
```
Comment (russgold) [inline idlj/src/main/java/com/sun/tools/corba/ee/idl/ValueEntry.java]: Previously, you changed a plural that used an apostrophe. Here you're adding one that does. Maybe use the same style consistently?

**Comment by russgold** [comment]
What a big job, here! I've noted some picky points.

**Comment by Cousjava** [comment]
@russgold I've made the requested changes

**Pull request merged**